app/angular/package.json

```diff
@@ -50,7 +50,7 @@
     "fork-ts-checker-webpack-plugin": "^4.1.6",
     "global": "^4.4.0",
     "postcss": "^7.0.35",
-    "postcss-loader": "npm:@phated/postcss-loader@4.2.0-phated.1",
+    "postcss-loader": "^4.1.0",
```

Comment:
??

Reply:
My thought with this was that requiring the peerDependency wouldn't need the `customPostcss` option because any top-level `postcss` would take precedence over the `@storybook/core` transitive dependency of postcss. Especially if my option wasn't going to be upstreamed...

It looks like they might be interested in taking the contribution after a rename, which I just submitted. So we will have to decide what we want here.

Reply:
I reverted this because the examples were failing to build. I'm not sure how they are setup differently than my local env 🤷